paimon-core/src/main/java/org/apache/paimon/schema/SchemaManager.java

```diff
@@ -58,22 +60,26 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import static org.apache.paimon.catalog.Catalog.DB_SUFFIX;
 import static org.apache.paimon.utils.FileUtils.listVersionedFiles;
 import static org.apache.paimon.utils.Preconditions.checkState;
 
 /** Schema Manager to manage schema versions. */
 public class SchemaManager implements Serializable {
 
     private static final String SCHEMA_PREFIX = "schema-";
+    private static final String UNKNOWN_DATABASE = "unknown";
 
     private final FileIO fileIO;
     private final Path tableRoot;
+    private final Identifier tableIdentifier;
```

Comment:
Can this be a local field? Because this unknown identifier is not correct, we can hide it in exception.

Reply:
done.